libs/portals/admin/core/src/lib/paths.ts

```diff
@@ -0,0 +1,22 @@
+export enum AdminPortalCorePaths {
```

Comment:
We don't want to maintain a global paths object like before. Unless you discussed something different in the office.

These path objects should be inside each module instead.